### DISCUSSION TITLE
May| Ambassdor Program Expense Details

### DISCUSSION THREAD

**Discussion by Cindy884** [ringdao-community-guild/discussion#17] [Show and tell]
The work submitted by the ambassador in dework has been reviewed. 
### Total Expense
2025 USD(equal to 562500 Ring) 
### DCDAO Recap
Click [here](https://dcdao.notion.site/May-Darwinia-Community-DAO-Recap-36f9d52eb5084c19a3703b6819673168?pvs=4) to learn more!

**Comment by hackfisher** [comment]
> Total Expense
> 2023 USD(equal to 562500 Ring)

What is the expense for? Including all the proposals in DCDAO, or for the position of Community Manager? or including Ambassador Program?

**Reply by Cindy884** [reply]
This expense is only about the ambassador program.

**Reply by hackfisher** [reply]
The title in the link need to be changed too.

**Comment by hackfisher** [comment]
> DCDAO Recap
> Click [here](https://dcdao.notion.site/May-Darwinia-Community-DAO-Recap-36f9d52eb5084c19a3703b6819673168?pvs=4) to learn more!

We can see that there are some datas screenshots and resources links, looking forward that there could be some description and summary the key results and progress which can be easily read as a "Recap".

Thanks for the updates! Looking forward to the monthly report for DCDAO and the position of Community Manager too.

**Comment by hackfisher** [comment]
And one more question, it looks like the details for this month is not following the proposal in https://snapshot.org/#/darwiniacommunitydao.eth/proposal/0x69bc76854f78a4d995b6ae47d27c95e12424c5aab60261238757575daa04e61d

by assigning two different groups of roles to ambassadors including "Ambassador" and "Senior Ambassdaor", could you confirm that we will still following the proposal or it will be started and effective from June?

**Reply by Cindy884** [reply]
Thks. As we communicated in the 5/31 DCDAO Meeting, this proposal vote time ended on 31/5 so the proposal will be effective from June's ambassador program.  The June Ambassador Program Expense will follow this proposal.
